letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -1107,7 +1117,7 @@ if [ "$1" = "--le-auto-phase2" ]; then
     fi
 
     if [ -f "$VENV_BIN/letsencrypt" -a "$INSTALL_ONLY" != 1 ]; then
-      error "Certbot will no longer receive updates."
+      error "certbot-auto will no longer receive updates and may stop working in the future."
```

Comment:
I do not know if it is relevant, but I would like to make this message even more extra clear. We still see several people mixing `certbot-auto` with `certbot`. Maybe we should say something more explicit about the fact that certbot-auto is the installation method, not certbot itself?